docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -63,21 +63,39 @@ If the battery is low, the connection between your station and the M5 gateway wi
 
 ## Sensor Problems
 
-Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
-There are several reasons that you may experience this error, which includes:
+We have discovered that the weather data packets received, did not meet our quality standards. 
+To ensure the accuracy of the information sent by our weather stations, we evaluate all data packets using our [Quality of Data (QoD) algorithm](/project/quality-of-data). For more information about this algorithm, please refer to this [here](/project/quality-of-data). 
+
+Below are some of the possible errors and the appropriate actions you can take to resolve the issues:
+
+
+#### Out of Bounds Data 
+It's possible that a sensor sends a completely irrational value, which implies to a faulty sensor. It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com)  for sensor replacement.
```

Comment:
`It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com) for sensor replacement.`

If this error persists for multiple days, user must first do a complete power cycle.
1. Remove batteries
2. Cover the solar panel so the station isnt powered by the panel while the batteries are out.
3. Press the reset button a few times to drain all stored power
4. Wait a few minutes
5. Put batteries in

If the above doesn't work, then contact support.